python/ray/rllib/a3c/tfpolicy.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+import ray
+import gym
+from ray.rllib.a3c.policy import Policy
+
+
+class TFPolicy(Policy):
```

Comment:
Eventually we will want to move this to the top level RLlib dir but I guess we need to do more refactoring for that.